vita3k/gui/src/information_bar.cpp

```diff
@@ -379,7 +403,7 @@ static void draw_notice_info(GuiState &gui, EmuEnvState &emuenv) {
     }
 
     if (ImGui::IsWindowHovered(ImGuiHoveredFlags_RootWindow) && !ImGui::IsAnyItemHovered() && ImGui::IsMouseClicked(ImGuiMouseButton_Left)) {
-        if (notice_info_state) {
+        if (notice_info_state && (notice_info_count_new > 0)) {
```

Comment:
```suggestion
        if (notice_info_state && notice_info_count_new > 0) {
```

Reply:
for that is normal, is formating code of vit'a3k
when compare have more one condition, need ading ( ) if conjdinition have two things with compare operator

Reply:
This isn't part of code formatting. Without these parenthesis, the formatter doesn't complain, no warning is generated and the code is still right as comparisons always have priority over logical operators, but anyway.